lua/lspconfig/server_configurations/lsp_ai.lua

```diff
@@ -0,0 +1,21 @@
+return {
+  default_config = {
+    cmd = { 'lsp-ai' },
+    filetypes = {},
+    root_dir = nil,
+    single_file_support = true,
+    init_options = {
+      memory = {
+        file_store = vim.fn.json_decode("{}"),
```

Comment:
Fix for the lint issue:

```suggestion
        file_store = vim.fn.json_decode '{}',
```

Reply:
I did not notice that, that's fixed thanks.